### ISSUE TITLE
condition not found

### ISSUE THREAD

**Issue by maddin200** [go-critic/go-critic#1216]
found:

```go
package test03

func test03() {
    var sum int
    for ii := 0; ii > 100; ii++ {
        sum += ii
    }
}
```

```
badCond: `ii > 100` in loop; probably meant `ii < 100`?
```

not found:

```go
package test03

func test03() {
    var sum int
    for ii := 0; 100 < ii; ii++ {
        sum += ii
    }
}
```

**Comment by quasilyte** [comment]
Hello.

I'm not sure I understand what kind of issue you're reporting.

**Comment by maddin200** [comment]
it is the same condition - should be also mentioned by go-critic

**Comment by quasilyte** [comment]
Ah, now I see. It was quite hard to notice the difference in there!
Makes sense, I guess!

**Comment by cristaloleg** [comment]
Fixed in #1287
